app/src/androidTest/java/com/github/se/polyfit/ui/screen/IngredientTest.kt

```diff
@@ -331,12 +331,12 @@ class IngredientTest : TestCase() {
   @Ignore("No Potential Ingredients Yet")
```

Comment:
Delete all ignored tests, as they're never going to be used after the code has been deleted

Reply:
yes forgot about those thanks for reminding me !